test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1570,3 +1570,84 @@ UCS_TEST_P(test_ucp_wireup_keepalive, attr) {
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_wireup_keepalive)
+
+class test_ucp_address_v2 : public test_ucp_wireup {
+public:
+    static void get_test_variants(std::vector<ucp_test_variant>& variants)
+    {
+        add_variant_with_value(variants, UCP_FEATURE_TAG,
+                               TEST_TAG | WORKER_ADDR_V2, "tag");
+        add_variant_with_value(variants, UCP_FEATURE_TAG,
+                               TEST_TAG | WORKER_ADDR_V2 | UNIFIED_MODE,
+                               "tag,unified");
+    }
+};
+
+UCS_TEST_P(test_ucp_address_v2, pack_iface_attrs) {
+    ucs_status_t status;
+    size_t size;
+    void *buffer;
+
+    if (has_transport("tcp")) {
+        UCS_TEST_SKIP_R("skipp checking tcp performance");
```

Comment:
`skipp` -> `skip`

Reply:
btw, why are we skipping TCP performance checking?

Reply:
it does not fit to our latency and bandwidth ranges defined for fp8 types and is unpacked to min/max values

Reply:
thanks!
shall we fix it in the future?
maybe add TODO and comment why we are skipping it for now?